src/content/developers/docs/nodes-and-clients/run-a-node/index.md

```diff
@@ -97,28 +104,69 @@ Before starting Ethereum client software, perform a last check that your environ
+Run your client on a testnet first to help make sure everything is working correctly. You need to declare any client settings that aren't default at the start.
+In order to connect to a consensus client the execution client must generate a `jwtsecret` at a known path. This path must be known by both clients as it is used
+to authenticate a local RPC connection between them. The execution client must also define a listening port for authenticated APIs.
+
+There are many ways to configure the execution client. You can use flags or the config file to declare your preferred configuration. Check out your client's documentation for the specific details.
+
 Client execution will initiate its core functions, chosen endpoints, and start looking for peers. After successfully discovering peers, the client starts synchronization. Current blockchain data will be available once the client is successfully synced to the current state.
 
-### Using the client {#using-the-client}
+### Getting the consensus client {#getting-the-consensus-client}
+
+There are currently five consensus clients to choose from. These are:
+
+[Lighthouse](https://lighthouse-book.sigmaprime.io/): written in Rust
+
+[Nimbus](https://nimbus.team/): written in Nim
+
+[Prysm](https://docs.prylabs.network/docs/getting-started/): written in Go
+
+[Teku](https://pegasys.tech/teku): written in Java
 
-Clients offer RPC API endpoints that you can use to control the client and interact with the Ethereum network in various ways:
+There is currently a [client diversity][client-div-link] issue where a large dominance of Prysm clients poses a risk
+to the health of the network. In response to the initial drive to even out the client diversity many Prysm nodes switched
+to Lighthouse to the extent that it now also has a problematic market share. It is therefore recommended to consider choosing
+a minority client.
+
+There are several ways to download and install the consensus clients including prebuilt binaries, docker containers
+or building from source. Instructions for each client are provided in the documentation linked in the client list above.
+Users can choose the method that is right for them.
+
+### Starting the consensus client {#starting-the-consensus-client}
+
+The consensus client must be started with the right port configuration to establish a local RPC connection to the execution
+client. The consensus clients all have a command similar to `--http-webprovider` that takes the exposed execution
+client port as an argument.
+
+The consensus client also needs the path to the execution client's `jwt-secret` in order to authenticate the RPC
+connection between them. Each consensus client has a command similar to `--jwt-secret` that takes the file
+path as an argument. This must be consistent with the `jwtsecret` path provided to the execution client.
```

Comment:
Similar to above comment, just want to make sure it's safe for people to do this if their plan is to run Mainnet (especially if staking).

We may want to consider a note here saying that this step should be performed once client have been updated with the TTD for The Merge (which of course Mainnet will be the last one to wait for)

Reply:
yea agree, play it safe